### PR TITLE
Add test case for default `git commit`

### PR DESCRIPTION
**Pull request by russellkan**
Addressing Issue #5 

Added ability to test hooks when running default `git commit` by creating a script and exporting it as the temporary `EDITOR` for `git commit` to call.

### CODE REVIEW

**Comment by russellkan** [comment]
The interesting part is that the hooks actually worked for commit 3981163.

And the `commit-msg` hook works too for commit c28dba2 where `merge` is already in the commit message.

File: tests/hook_test.py [hunk 1]
```diff
@@ -31,6 +31,25 @@ def tmp_repo(tmpdir):
     subprocess.call(['rm', '-rf', directory])
 
 
+@pytest.fixture
```
Comment (johndgiese) [inline tests/hook_test.py]: Very nice :)

File: tests/hook_test.py [hunk 2]
```diff
@@ -31,6 +31,25 @@ def tmp_repo(tmpdir):
...
+    # create EDITOR executable and set environment variable
+    editor_path = os.path.join(str(tmpdir), 'tmp_editor.sh')
+    with open(editor_path, 'w') as f:
+        f.write('#!/bin/bash\n')
+        f.close()
+    subprocess.call(['chmod', '+x', editor_path])
+    old_editor = os.environ.get('EDITOR', None)
+    os.environ["EDITOR"] = editor_path
+
+    yield
```
Comment (johndgiese) [inline tests/hook_test.py]: I like this use of vertical whitespace to separate the construction from the destruction.

**Comment by johndgiese** [comment]
Looks great!

**Pull request merged**